src/aleph_client/commands/utils.py

```diff
@@ -291,28 +364,42 @@ async def update_table(queue: asyncio.Queue, table: Table):
             break
         table.add_row(*data)
 
-async def fetch_crn_system(session: aiohttp.ClientSession, node: NodeInfo) -> str:
+async def fetch_crn_system(session: aiohttp.ClientSession, node: NodeInfo) -> CpuProperties:
+    data = None
+
     try:
         async with async_timeout.timeout(settings.HTTP_REQUEST_TIMEOUT + settings.HTTP_REQUEST_TIMEOUT * 0.3 * random()):
             url: str = node["address"].rstrip('/') + '/about/usage/system'
             async with session.get(url) as resp:
                 resp.raise_for_status()
-                data = await resp.json()
+                data_raw = await resp.json()
+                data = MachineUsage.parse_obj(data_raw)
+                # print("data:", data)
+    except TimeoutError:
+        logger.debug(f'Timeout while fetching: {url}')
+    except aiohttp.client_exceptions.ClientConnectionError:
+        logger.debug(f'Error on connection: {url}')
     except Exception as e:
-        data = None
+        logger.debug(f'This error occured: {e}')
```

Comment:
It is usually a bad practice to catch every exception, unless the context is a service that must keep working. 

In this case, two known exceptions are already handled and other issues (memory full, ...) may not be handled correctly by the program anyways.
What if it is a "variable not defined" for example ? This should definitely not go unnoticed.